origin-dapp/src/actions/App.js

```diff
@@ -129,7 +129,10 @@ export function localizeApp() {
     }
   }
 
-  const messages = translations[bestAvailableLanguage]
+  const messages = translations[bestAvailableLanguage];
```

Comment:
Thanks for the PR @riverfor! It looks like this line is causing a test failure. You can see it by inspecting the TravisCI output, or by running `cd origin-dapp && npm run test`. It's just a formatting error caused by the extra semicolon because we are using ESLint for code formatting.

Reply:
Thanks @tomlinton 

I have merged the master and updated the code already, would you like to have a look when you are free ?